Casks/v/vibetunnel@beta.rb

```diff
@@ -0,0 +1,32 @@
+# typed: strict
+# frozen_string_literal: true
+
+cask "vibetunnel@beta" do
```

Comment:
```suggestion
cask "vibetunnel" do
```

When the "beta" version is the "most stable" version we don't append `@beta`, but just use the primary token.